scripts/make_gitrev.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+file=include/spl/sys/zfs_gitrev.h
```

Comment:
We should also add this to .gitignore

Reply:
Let's also add a comment explaining that this isn't done with a `zfs_gitrev.h.in` file since it needs to be generated for every invocation of `make`, not once during `configure`.